portfolio/src/main/webapp/index.html

```diff
@@ -7,13 +7,47 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="container"> 
+               
+            <div id="intro">
+        <h1>Erik Rodriguez</h1>
+            <p>
+                Hi everyone my name is Erik, 
+                I am a sophomore Software Engineering Student at The University of Texas at Dallas 
+                I am passionate about technology, leadership and designing products.  
+                My goal is to become a Software Engineer and have meaningful impact solving problems around the world.
+            </p>
+
+
+        <h2> About me </h2>  
+            <p> I was born in Dallas, TX, however I spent the majority of my childhood in a small village named "Ojitos" 
+                which is located in Zacatecas, Mexico. And while I was there I didn't know anything about Computer Science 
+                however, I did play a lot of video games and I dreamed of creating my own one day. 
+                When I arrived to the United States I tried different types of classes like, video production, welding and by
+                my senior year I decided to major in Software Engineering because I saw it as a way to incorporate problem solving skills
+                arts, and other fun things that I liked about my other classes. 
+                
+        <h2> Hobbies </h2>
+        <ul>
+            <li>Gaming</li>
+            <li>Photography</li>
+            <li>Video Editing</li>
+            <li>Learning Ukelele</li>
+            <li>Designing things in general</li>
+        </ul>      
+        </div>
+
+        <div id=photosofme>
+            <img src="/Images/IMG_6155.jpg"  height="250px"/>
```

Comment:
I think the leading "/" needs to be removed, from https://www.w3schools.com/html/html_filepaths.asp, 
"`src="images/picture.jpg` indicates The "picture.jpg" file is located in the images folder in the current folder".